src/components/composite/aspect/AspectForm/AspectForm.tsx

```diff
@@ -69,9 +68,16 @@ const AspectForm: FC<AspectFormProps> = ({
     },
   ];
 
+  const getDescriptionValue = () => {
+    if (!aspect) {
+      return '';
+    }
+    return aspect.description ?? descriptionTemplate ?? '';
```

Comment:
This way if you enter your own description, get back, choose another aspect type, then go next, the default description won't be reflected when you change the type.
This is more of a UX decision - we can leave it that way and return later.